app/modules/assets/schemas.py

```diff
@@ -30,14 +30,18 @@ class DetailedAssetSchema(BaseAssetSchema):
     Detailed Asset schema exposes all useful fields.
     """
 
+    # <TODO - MWS>
```

Comment:
There is a needed change (marked as `<TODO -MWS>`) to the asset schema to disentangle it with asset_groups.  The solution I have for now is to patch out the value from the schema.  Needs a permanent solution